src/Web/ViewModels/FraudProtectionIOModel.cs

```diff
@@ -37,7 +37,8 @@ public void Add(object request, SampleResponseRaw response, string name = "", bo
             {
                 Request = skipSerialization ? request as string : JsonSerializer.Serialize(request, JsonSerializerOptions),
                 Response = JsonSerializer.Serialize(response.RawData, JsonSerializerOptions),
-                Name = name
+                Name = name,
+                CallTime = response.ResponseTime
```

Comment:
nit: could we call this variable `ResponseTime`?

Reply:
updated!